### PR TITLE
Abcd class is added

### PR DESCRIPTION
**Pull request by asranjith**
Abcd testCase added

### CODE REVIEW

File: src/test/java/DummyPackage/Abcd.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package DummyPackage;
+
+public class Abcd {
+  public static void main(String[] args) {
+	System.out.println("AS ranjith");
```
Comment (mugilanmonish) [inline src/test/java/DummyPackage/Abcd.java]: Remove sout statment.

File: src/test/java/DummyPackage/Abcd.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package DummyPackage;
+
+public class Abcd {
```
Comment (mugilanmonish) [inline src/test/java/DummyPackage/Abcd.java]: Change the class name as per standards

**Review by mugilanmonish — CHANGES_REQUESTED** [review]

File: src/test/java/DummyPackage/Abcd.java (hunk 2 above)
Comment (asranjith) [reply src/test/java/DummyPackage/Abcd.java]: Abcd name changed to TestCaseName

File: src/test/java/DummyPackage/Abcd.java (hunk 1 above)
Comment (asranjith) [reply src/test/java/DummyPackage/Abcd.java]: removed sout statement

**Review by mugilanmonish — APPROVED** [review]

**Pull request merged**